src/lib/viewers/image/MultiImage.scss

```diff
@@ -33,7 +33,8 @@
 .bp-images {
     border: 0 none;
     margin: 0 auto;
-    padding: 0;
+    padding: 7px;
```

Comment:
Can you add a comment explaining that this is to allow the focus treatment to be visible?